src/modules/remove_completed.js

```diff
@@ -0,0 +1,15 @@
+import { taskList, completedElements, taskArr } from './variables.js';
+
+export default function removeAllCompleted() {
+  while (completedElements.length !== 0) {
+    taskArr.splice(completedElements[0].id, 1);
+    completedElements[0].remove();
+
+    taskArr.forEach((task, i) => {
+      task.index = i;
+      taskList.children[i].id = i;
+    });
+  }
```

Comment:
- [ ] Please, consider using the `filter()` method implementation of a function for the "Clear all completed" button, as it is required in the project. With the `forEach` version, need to analyze that there is an empty array defined first and then elements are added to it based on the condition. With the `filter()` function, we only need to concentrate on the test function logic. You can read more [here](https://medium.com/front-end-weekly/stop-array-foreach-and-start-using-filter-map-some-reduce-functions-298b4dabfa09) about `filter` and `map` methods

Reply:
@elmar8287

Reply:
> 

Dear @Jord4N-AC could you implement the filter() method as required in the project? I think you can do it easily 👍